internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -67,26 +69,26 @@ type AddonVersion struct {
 type AddonsVersion map[Addon]*AddonVersion
 
 type KubernetesVersion struct {
-	ControlPlaneComponentsVersion ControlPlaneComponentsVersion
-	ComponentsVersion             ComponentsVersion
-	AddonsVersion                 AddonsVersion
+	ComponentHostVersion      ComponentHostVersion
+	ComponentContainerVersion ComponentContainerVersion
+	AddonsVersion             AddonsVersion
 }
 
 type KubernetesVersions map[string]KubernetesVersion
 
 var (
-	Versions = KubernetesVersions{
+	supportVersions = KubernetesVersions{
```

Comment:
I don't see a strong need for renaming this (aside from not making it public), it also makes the whole diff bigger. However, I think it's fine, but I'd use `supportedVersions` instead, WDYT?

Reply:
Sure, I made it private since the originally same package uses this variable directly, some use API to access it. Will change to `supportedVersions`.